api-configuration/src/main/java/org/jvalue/ods/apiConfigurationService/endpoint/ApiConfigurationService.java

```diff
@@ -0,0 +1,143 @@
+package org.jvalue.ods.apiConfigurationService.endpoint;
+
+import org.jvalue.ods.apiConfigurationService.helper.GraphqlQueryFactory;
+import org.jvalue.ods.apiConfigurationService.helper.SqlFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Arrays;
+
+import static org.jvalue.ods.apiConfigurationService.helper.GraphqlQueryFactory.createSQLGraphQuery;
+
+@RestController
+@RequestMapping("/")
+public class ApiConfigurationService implements ApiConfigurationServiceI {
+
+  private static final String HASURA_ADMIN_ENDPOINT = "http://172.17.0.1:7654/v1/query";
+  private static final String DEFAULT_SCHEMA = "storage";
+
+  @PostMapping("test")
+  public HttpStatus testThis() {
+    new SqlFactory().createViewQueryWithUnpackedFields("3", "s");
+    return HttpStatus.CONFLICT;
+  }
+
+  @Override
+  @PostMapping("/deleteAPIForTable")
+  public HttpStatus deleteAPIForTable(@RequestParam(name = "tableName", required = true) String tableName) {
```

Comment:
Are the table methods used from outside?

Reply:
They used to be before i switched to communicating via rabbitmq